package.json

```diff
@@ -103,10 +101,12 @@
     "apollo-server-express": "^2.24.1",
     "apollo-server-express-v3": "npm:apollo-server-express@3.6.2",
     "async": "^3.2.2",
+    "semver": "7.5.4",
     "aws-sdk": "2.1450.0",
     "basic-auth-parser": "0.0.2",
     "bluebird": "^3.7.2",
     "body-parser": "^1.19.0",
+    "bson": "^4.7.2",
```

Comment:
FYI: Had to install bson 4 to fix typescript

>node_modules/@types/mongodb/index.d.ts:58:74 - error TS2724: '"bson"' has no exported member named 'ObjectID'. Did you mean 'ObjectId'?

mongodb v6 uses bson v6
@types/mongodb is deprecated but still used by mongoose-v5@npm:mongoose@5.13.17

That's why we need to pin bson to v4 on the root